contracts/EverlongStrategyFactory.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { EverlongStrategy } from "./EverlongStrategy.sol";
+import { IEverlongStrategy } from "./interfaces/IEverlongStrategy.sol";
+import { IEverlongStrategyFactory } from "./interfaces/IEverlongStrategyFactory.sol";
+import { EVERLONG_KIND, EVERLONG_VERSION } from "./libraries/Constants.sol";
+
+/// @author DELV
+/// @title EverlongStrategyFactory
+/// @notice A factory for creating Everlong instances.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract EverlongStrategyFactory is IEverlongStrategyFactory {
+    /// @inheritdoc IEverlongStrategyFactory
+    string public constant name = "EverlongStrategyFactory";
+
+    /// @inheritdoc IEverlongStrategyFactory
+    string public constant kind = EVERLONG_KIND;
+
+    /// @inheritdoc IEverlongStrategyFactory
+    string public constant version = EVERLONG_VERSION;
+
+    address public immutable emergencyAdmin;
+    address public management;
+    address public performanceFeeRecipient;
+    address public keeper;
+
+    /// @notice Track the deployments. asset => pool => strategy
+    mapping(address => address) public deployments;
+
+    constructor(
```

Comment:
Nit: Missing natspec

Reply:
This file is missing a lot of documentation and doesn't follow our style guide. Can we update this so that it looks like all of our other code?